libcontainer/container_linux.go

```diff
@@ -1021,3 +1025,75 @@ func (c *linuxContainer) currentState() (*State, error) {
 	}
 	return state, nil
 }
+
+const (
```

Comment:
Can we move this message code outside of this file and into maybe a `message.go` file?  This file is already pretty large.